Emerald.CoreX/Class1.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Emerald.CoreX
```

Comment:
The idea of a Core project is that it contains common files across SDKs and implementation projects. I don't understand why we can't just add it's contents to `Emerald.Core` anyway 😕
The name CoreX is in the name of bad form in the first place.. just using Core or calling it OneCore would be better

Reply:
The current core is bit outdated and un-organized. I think we should rewrite the Core with new features and a well-organized code. 
It's so hard to replace the current thing at once because every class/thing of `Emerald.Core` depends on another class/thing. That's why I created `Emerald.CoreX`
I also wanted to remove that `Localized.xxxx` thing and `TaskHelper`, because it's kind of thing that should be done by the UI